.github/workflows/lint.yml

```
@@ -4,8 +4,17 @@ on: [push, pull_request, workflow_dispatch]
 
 jobs:
   pre-commit:
+    name: Run Pre-Commit
```

The official tool name is lower case: https://pre-commit.com/
```suggestion
    name: Run pre-commit
```

---

Its not particularly critical, but this was intended to avoid confusion and ambiguity between "Pre-Commit" the tool (and its hooks) and "pre-commit" the Git stage (and its hooks).

To note, the tool author writes _everything_ in lower-case, including the names of other formats and tools whose official (and widely used) spellings are uppercase/titlecase, e.g. JSON, TOML, XML, YAML, etc. (and has [adamantly refused](https://github.com/pre-commit/pre-commit-hooks/pull/634) to correct them on [multiple occasions](https://github.com/pre-commit/pre-commit-hooks/pull/413) despite [many requests](https://github.com/pre-commit/pre-commit-hooks/pull/670)), so I don't really see an issue with adopting Proper Case here to reduce ambiguity, given we have reasons for it here beyond personal pique.

That being said, its not _that_ important, so if you still see it differently I'll just go ahead and apply it.

---

Thanks, generally (and in this case) I think it would be better to spell things the way the author does, like for pytest and blurb. I don't think we really need to worry about ambiguity here.